task_manager/test_module.hh

```diff
@@ -0,0 +1,17 @@
+#include "task_manager.hh"
```

Comment:
nit: add copyright boilerplate and `#pragma once` at the top

Also, a FIXME comment for `#ifndef SCYLLA_BUILD_RELEASE`